projects/vtk.cmake

```diff
@@ -30,7 +30,7 @@ superbuild_add_project(vtk
     -DVTK_MODULE_ENABLE_VTK_FiltersGeneral=YES
     -DVTK_MODULE_ENABLE_VTK_FiltersGeometry=YES
     -DVTK_MODULE_ENABLE_VTK_IOCityGML=YES
-    -DVTK_MODULE_ENABLE_VTK_IOExodus=YES
+    -DVTK_MODULE_ENABLE_VTK_IOExodus=NO
```

Comment:
`IOExodus` does not build with `musl` because it relies on an extension file of the glibc (`execinfo.h`) through hdf5

Reply:
there is a few ways to fix this. 1. remove the execinfo dependency, 2. do not use musl, 3. do not build IOExodus for the python wheels.

3. is trivial, is it what you had in mind ?

Reply:
I think the execinfo issue was also the issue when building VTK for Android. I'd say it would be great if it's fixed on VTK side.  
Regarding musl, `cibuildwheel` is generating wheels for `manylinux` and `musl` for maximum compatibility. We can drop `musl` entirely but that should not be our first option.  
Excluding IOExodus can be good enough if fixing the issue in VTK is too hard.

Reply:
I'd say let's add a CMake variable to disable exodus for now. It would be useful anyway if people don't want Exodus and want to reduce the size of the binaries.

Reply:
> IOExodus does not build with musl because it relies on an extension file of the glibc (execinfo.h) through hdf5
 
 I do not see an usage of execinfo.h in hdf5 or any deps of IOExodus

Reply:
The only deps that uses execinfo is KWSys, but this can be fixed with `KWSYS_NO_EXECINFO`.

Reply:
https://github.com/Kitware/VTK/blob/d4eda446cf8efe859cb507c84ba7b307b6e24311/ThirdParty/netcdf/vtknetcdf/libhdf5/hdf5debug.c#L9

Reply:
IOExodus does not depend on netcdf though

Reply:
ThirdParty exodus , found it, thanks

Reply:
actually, its not used: 

```
#undef HAVE_EXECINFO_H
```

from: 

https://github.com/Kitware/VTK/blob/d4eda446cf8efe859cb507c84ba7b307b6e24311/ThirdParty/netcdf/vtknetcdf/config.h.in#L150

Reply:
So I tried removing execinfo from my system and enabling IOExodus, I only needed to fix KWSys to be able to build.

Reply:
I'm pretty sure the error was in H5debug.c

Reply:
I will need the actual error, I'm not reproducing anything related.

Reply:
you can try to build VTK on an Alpine based Docker image.

Reply:
I just did, without issue:

```
apk install build-base vim cmake git linux-headers
git clone https://gitlab.kitware.com/vtk/vtk.git
cd vtk
mkdir build
cd build
cmake -DVTK_GROUP_ENABLE_Imaging=DONT_WANT -DVTK_GROUP_ENABLE_MPI=DONT_WANT -DVTK_GROUP_ENABLE_Qt=DONT_WANT -DVTK_GROUP_ENABLE_Rendering=DONT_WANT -DVTK_GROUP_ENABLE_StandAlone=DONT_WANT -DVTK_GROUP_ENABLE_Views=DONT_WANT -DVTK_GROUP_ENABLE_Web=DONT_WANT -DVTK_ENABLE_LOGGING=OFF -DVTK_MODULE_ENABLE_VTK_IOExodus=YES ../
make
```

Reply:
I'll try again later.